filecoin-proofs/examples/ffi/main.rs

```diff
@@ -48,14 +50,21 @@ unsafe fn create_and_add_piece(
     let c_piece_key = rust_str_to_c_str(piece_key.clone());
     defer!(free_c_str(c_piece_key));
 
+    // write piece bytes to a temporary file
+    let mut file = NamedTempFile::new().unwrap();
+    let p = { String::from(file.path().to_str().unwrap().clone()) };
```

Comment:
- this should definitely not `unwrap`, at least please use `.expect("the reason I think this will never happen")`
- you can just do  as far as I can tell
```rust
let p = file.path().to_str().to_string()
```

Reply:
Okay, okay, I won't unwrap - even in test code. Will update to use `expect` and `expects` with a nice error message.